### ISSUE TITLE
Scratch Link does not work with Bookworm

### ISSUE THREAD

**Issue by ralf-krause** [puavo-org/puavo-os#636]
Hey, 
I found that Scratch Link (pyscrlink) does not start anymore with the current Puavo Bookworm image opinsys-os-opinsys-bookworm-2023-11-03-091807-amd64. 

I wanted to start the Scratch extension for micro:bit https://scratch.mit.edu/microbit. This extension allows to conrol the micro:bit with Scratch via Bluetooth. It's a kind of direct mode ... the Scratch program is running on the computer and the micro:bit has an interpreter for the recieved commands. Yes, I tested that Bluetooth is working with Bookworm on my test device md-lap-3011.opinsys.fi.

The Scratch Link for Linux is https://github.com/kawasaki/pyscrlink.
Yes, Scratch Link is working with the Puavo Bullseye image opinsys-os-opinsys-bullseye-2023-11-02-113644-amd64 (Roma).

Best regards,
Ralf

**Comment by ralf-krause** [comment]
Yes, the BIOS 1.60 of my test device https://magdeburg.opinsys.fi/devices/502444/devices/556178 is the newest one.

**Comment by ateras** [comment]
The corresponding issue in the pyscrlink project: https://github.com/kawasaki/pyscrlink/issues/41 
When the problem with Debian 12 is resolved it'll most likely solve the issue also in Puavo.

**Comment by ralf-krause** [comment]
Could we please try this for Puavo Bookworm? https://github.com/kawasaki/pyscrlink/pull/44

**Comment by ralf-krause** [comment]
Scratch Link still does not work for Puavo Bookworm and it will not work for Puavo Trixie. The software is developed for Debian 11 (Bullseye) and does not work for Debian 12 (Bookworm) and Debian 13 (Trixie). The code is not changed since 3 years.

You should Scratch Link from the Puavo repository and from the PuavoConfig.
